src/main/java/at/hannibal2/skyhanni/test/DebugCommand.kt

```diff
@@ -102,7 +103,7 @@ object DebugCommand {
             event.addData("not on SkyBlock, but on Hypixel")
             return
         }
-        if (LorenzUtils.skyBlockIsland == IslandType.UNKNOWN) {
+        if (IslandType.UNKNOWN.isInIsland()) {
```

Comment:
Im against this change here. its not a real island, making the wording "is in island" wrong. lets keep the original islandtype == unknown here